<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BareMetal::Internal::BareMetalDeviceConfigurationWizardSetupPage</class>
 <widget class="QWidget" name="BareMetal::Internal::BareMetalDeviceConfigurationWizardSetupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="namelabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="hostNameLineEdit">
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="HostPortLabel">
     <property name="text">
      <string>GDB port:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="hostLabel">
     <property name="text">
      <string>GDB host:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="portSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>3333</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="gdbInitLabel">
     <property name="text">
      <string>GDB commands:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPlainTextEdit" name="gdbInitCommandsPlainTextEdit">
     <property name="plainText">
      <string>load
monitor reset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
